job-server/src/main/scala/spark/jobserver/HealthCheck.scala

```diff
@@ -0,0 +1,81 @@
+package spark.jobserver
+
+import akka.actor.ActorRef
+import akka.pattern.ask
+import akka.util.Timeout
+import org.slf4j.LoggerFactory
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.util.control.NonFatal
+import spray.httpx.SprayJsonSupport.sprayJsonMarshaller
+import spray.json.DefaultJsonProtocol._
+import spray.http.StatusCodes
+import spray.routing.{HttpService, RequestContext}
+import spark.jobserver.io.JobInfo
+import spark.jobserver.io.JobDAOActor
+
+trait HealthCheckService {
```

Comment:
nit:
`HealthCheck` is enough (in my opinion)

Reply:
Addressed